### PR TITLE
Quick fix to prevent group admin to delete an user

### PR DESCRIPTION
**Pull request by hugocollignon**
Simple patch for https://github.com/owncloud/core/issues/3362

### CODE REVIEW

File: settings/js/users/users.js [hunk 1]
```diff
@@ -116,16 +116,18 @@ var UserList = {
...
-			var deleteImage = $('<img class="svg action">').attr({
-				src: OC.imagePath('core', 'actions/delete')
-			});
-			var deleteLink = $('<a class="action delete">')
-				.attr({ href: '#', 'original-title': t('settings', 'Delete')})
-				.append(deleteImage);
-			$tr.find('td.remove').append(deleteLink);
-		} else if (OC.currentUser === user.name) {
-			$tr.find('td.remove a').remove();
+		if (OC.currentUser === 'admin') {
```
Comment (LukasReschke) [inline settings/js/users/users.js]: Can you elaborate on the portability of this code?

This only checks whether the currently logged-in user is named "admin". I doubt that this is what we want to do here :wink:

**Comment by MorrisJobke** [comment]
Better view of the diff https://github.com/owncloud/core/pull/22793/files?w=1

File: settings/js/users/users.js (hunk 1 above)
Comment (PVince81) [inline settings/js/users/users.js]: To check for admin permission in the JS code, use `OC.isUserAdmin()`.

**Comment by PVince81** [comment]
The ticket https://github.com/owncloud/core/issues/3362 is a feature request to make subadmin permissions tweakable, which would imply adding config switches in some places and possibly a UI for admins to configure what subadmin are allowed to do.

While this might be a quick fix, this short additional will change the default behavior without providing a way to bring it back.

I'm closing this due to lack of feedback.
Feel free to reopen or submit another pull request. Maybe adding a config switch would be a more acceptable solution.

Thank you for your understanding and also for your interest in contributing !

**Pull request closed without merging**